org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/JavaLanguageServerPlugin.java

```diff
@@ -206,6 +218,38 @@ public IProxyService getProxyService() {
 		return null;
 	}
 
```

Comment:
i thought that only `org.eclipse.jdt.ls.core` has the plugin dependencies for jdt, since `org.eclipse.jdt.ls.debug` already introduced the dependencies, it's ok to move there